installation/language/en-GB/joomla.ini

```
@@ -283,3 +283,14 @@ JLIB_FORM_CONTAINS_INVALID_FIELDS="The form cannot be submitted as it's missing
 JLIB_FORM_FIELD_INVALID_VALUE="This value is not valid."
 JLIB_FORM_FIELD_REQUIRED_CHECK="One of the options must be selected."
 JLIB_FORM_FIELD_REQUIRED_VALUE="Please fill in this field."
+
+; Installation progress modal
+INSTL_PROGRESS_MODAL="Installation Progress"
+INSTL_PROGRESS_STEP_DBCHECK="Check DB credentials"
+INSTL_PROGRESS_STEP_CREATE="Check system configuration"
+INSTL_PROGRESS_STEP_POPULATE1="Install basic database tables"
+INSTL_PROGRESS_STEP_POPULATE2="Install support database tables"
+INSTL_PROGRESS_STEP_POPULATE3="Install extensions tables"
+INSTL_PROGRESS_STEP_CUSTOM1="Install localisations"
+INSTL_PROGRESS_STEP_CUSTOM2="Install customisations"
+INSTL_PROGRESS_STEP_CONFIG="Saving configuration and clean-up"
```

```suggestion
INSTL_PROGRESS_STEP_CONFIG="Saving configuration and cleaning up"
```

Otherwise it suggests that it is saving a clean-up